acceptance/tests/agent/agent_disable_lockfile.rb

```diff
@@ -1,100 +1,83 @@
-test_name "C4553 - agent --disable/--enable functionality should manage the agent lockfile properly"
-confine :except, :platform => 'cisco_nexus' #See BKR-749
+test_name "C4553 - agent --disable/--enable functionality should manage the agent lockfile properly" do
+  confine :except, platform: "cisco_nexus" # See BKR-749
 
-tag 'audit:integration', # lockfile uses the standard `vardir` location to store/query lockfile.
-                         # The validation of the `vardir` at the OS level
-                         # should be accomplished in another test.
-    'audit:medium',
-    'audit:refactor'     # This test should not require a master. Remove the use of `with_puppet_running_on`.
+  require "json"
 
-#
-# This test is intended to ensure that puppet agent --enable/--disable
-#  work properly, both in terms of complying with our public "API" around
-#  lockfile semantics ( http://links.puppet.com/agent_lockfiles ), and
-#  in terms of actually restricting or allowing new agent runs to begin.
-#
+  tag 'audit:integration', # lockfile uses the standard `vardir` location to store/query lockfile.
+                           # The validation of the `vardir` at the OS level
+                           # should be accomplished in another test.
+      'audit:medium'
 
-require 'puppet/acceptance/temp_file_utils'
+  #
+  # This test is intended to ensure that puppet agent --enable/--disable
+  #  work properly, both in terms of complying with our public "API" around
+  #  lockfile semantics ( http://links.puppet.com/agent_lockfiles ), and
```

Comment:
This link does not appear to work

Reply:
Sadly all of our `http://links.puppet.com/*` are broken. Previously we used google links shortener to redirect to different "topic" pages. We're having a related discussion on another PR (/cc @jbondpdx). So for this PR I wouldn't add any more references to them, and we need to figure out next steps for the existing links in puppet code.

Reply:
Thanks @jtappa , I have removed the reference in an additional commit to this PR.